web-client/src/components/CartItem.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@/components/ui/button';
+import { Card, CardHeader, CardContent, CardFooter } from '@/components/ui/card';
+import { Trash2 } from "lucide-react";
+
+interface CartItemProps {
+    sockId: number;
```

Comment:
Instead of passing the individual pieces of an "item", just pass the item directly like this:

```ts
interface CartItemProps {
  item: CartItemType;
  onIncrease: () => void;
  onDecrease: () => void;
  onRemove: () => void;
}
```

Reply:
Fixed!

Reply:
@sebastian-nunez Would I then need to change lines 14-18 to just item?
<img width="452" alt="image" src="https://github.com/user-attachments/assets/194bf00d-3fee-477e-8aa8-130da6ed523d">
Should I also add const { name, size, price, quantity, imageUrl } = item;

Reply:
I think probably just getting the item and doing item.WHATEVER should be good